### PR TITLE
smoke b2c pdp scenario

### PR DESCRIPTION
**Pull request by khushal-rathore-metacube**
https://spryker.atlassian.net/browse/CC-21837

Written only add product to cart and add to wishlist scenario as remaining scenarios are covered in existing test cases in smoke b2c with below mentioned test cases names:

1. Product_labels
2. Product_PDP
3.  Product_Relations
4. Product_Sets
5. Product_Bundles
6. Catalog_Actions

### CODE REVIEW

File: tests/smoke/smoke_b2c.robot [hunk 1]
```diff
@@ -599,3 +599,9 @@ Refunds
     [Teardown]    Run keywords    Zed: login on Zed with provided credentials:    ${zed_admin_email}
     ...    AND    Zed: activate following discounts from Overview page:    Tu & Wed $5 off 5 or more    10% off $100+    20% off cameras    Tu & Wed €5 off 5 or more    10% off minimum order
  
+User_can_add_product_to_cart_and_wishlist_on_PDP
+    Yves: login on Yves with provided credentials:    ${yves_second_user_email}
+    Yves: go to PDP of the product with sku:    ${multi_variant_product_concrete_sku}
+    Yves: add product to the shopping cart
+    Yves: add product to wishlist:    My wishlist
```
Comment (AntonShopin) [inline tests/smoke/smoke_b2c.robot:606]: where are the assertions? How do you make sure that the right variant was added to the shopping cart and wishlist and is displayed there?

**Review by AntonShopin — CHANGES_REQUESTED** [review]
Could you please point me out which tests we already check: 
* User can see product groups (PDP)
* Verify PDP of a product set → User can add individual product to cart
* Verify PDP of a product: images, product description

File: tests/smoke/smoke_b2c.robot (hunk 1 above)
Comment (khushal-rathore-metacube) [reply tests/smoke/smoke_b2c.robot]: Done, fixed

**Comment by khushal-rathore-metacube** [comment]
Covered below scenarios:
User can see product groups (PDP)
Verify PDP of a product set → User can add individual product to cart
Verify PDP of a product: images, product description

**Pull request closed without merging**